src/components/Service/TestService.jsx

```diff
@@ -1,59 +1,164 @@
-import { useState, useEffect, useRef } from "react";
+import { useState, useEffect, useCallback } from "react";
 import { useNavigate } from "react-router-dom";
 import PropTypes from "prop-types";
 
 import Service from "../shared/Service/Service";
-
 import CONTENTS from "../../utils/ServiceContents";
+import { DEFAULT_WPM } from "../../utils/readTimeTest";
+import codeHighlighter from "../../utils/HighlightCodeBlock";
+
+const MIN_READ_TIME_FACTOR = 0.5;
+
+const calculateMinReadTime = (content) => {
+  const numOfWords = content.split(" ").length;
+  const baseTime = (numOfWords / DEFAULT_WPM) * 60;
+
+  return Math.max(Math.round(baseTime * MIN_READ_TIME_FACTOR), 2);
+};
 
 function TestService({ navigateNextPage }) {
-  const [isScrolledToBottom, setIsScrolledToBottom] = useState(false);
-  const testTextAreaRef = useRef(null);
+  const [currentContentIndex, setCurrentContentIndex] = useState(0);
+  const [buttonEnabled, setButtonEnabled] = useState(false);
```

Comment:
```suggestion
  const [isNextButtonEnabled, setIsNextButtonEnabled] = useState(false);
```

boolean 타입임을 명확히 하기 위해 is로 시작하면 좋을 것 같아요.

코드를 보니 이 상태값은 다음 버튼의 활성화 여부를 나타내는 것 같은데
좀 더 쉽게 코드를 파악할 수 있도록 변수명을 isNextButtonEnabled 또는 isNextButtonDisabled로 하는 것이 어떨지 의견 드립니다!